docs/running.md

```diff
@@ -121,11 +133,22 @@ MSSQL_DATABASE_DXTR={the name of the DXTR database}
 MSSQL_ENCRYPT={a string: true | false}
 MSSQL_TRUST_UNSIGNED_CERT={a string: true | false}
 
+ACMS_MSSQL_HOST={the FQDN of the ACMS SQL database}
+ACMS_MSSQL_DATABASE_DXTR={the name of the ACMS database}
+ACMS_MSSQL_ENCRYPT={a string: true | false}
+ACMS_MSSQL_TRUST_UNSIGNED_CERT={a string: true | false}
+
 # Required for SQL Auth. Required for connecting to SQL with SQL Identity.
 MSSQL_USER={the SQL Server Admin username}
 MSSQL_PASS={the SQL Server Admin user password}
+ACMS_MSSQL_USER={the ACMS SQL Server Admin username}
+ACMS_MSSQL_PASS={the ACMS SQL Server Admin user password}
 # Required for connecting to CAMS SQL server database with managed identity
 MSSQL_CLIENT_ID={OPTIONAL client id of Managed Identity with access}
+ACMS_MSSQL_CLIENT_ID={OPTIONAL client id of Managed Identity with access to the ACMS DB}
+
+AzureWebJobsStorage={Connection String to Azure Functions Storage account}
+MyTaskHub={Durable Functions Task Hub Name}
```

Comment:
Just a note that I have to put these in my `local.settings.json`. I'm not sure why it doesn't work in my `.env` file.